### ISSUE TITLE
Suppress console window 

### ISSUE THREAD

**Issue by e-dul** [madmaze/pytesseract#63]
Hi,
I found your project very useful and I like to thank you for your work :)
But after using it in standalone executable on windows, I'd like to suggest small improvement.
Currently (on win 8 and standalone exe) every call to pytesseract make console window to appear for ~1s which is very unpleasant. This small fix resolve this issue for me:   
```
sinfo = subprocess.STARTUPINFO()
sinfo.dwFlags = subprocess.CREATE_NEW_CONSOLE | subprocess.STARTF_USESHOWWINDOW
sinfo.wShowWindow = subprocess.SW_HIDE
proc = subprocess.Popen(command, stderr=subprocess.PIPE, startupinfo=sinfo)
```
Would you consider adding some setting(or better solution) to allow user to decide, if console should be visible?

**Comment by int3l** [comment]
@e-dul thanks for the suggestion.
The only problem is that this option is Windows specific.

So we will need to make it available only for the Windows platform.
I will hack on it, when I have time.

If someone else has a working fork with the proper changes, submit a PR.

**Comment by 2press** [comment]
To be able to run pytesseract from a standalone executable without a console created by pyinstaller I had to use ` proc = subprocess.Popen(command, **subprocess_args(True))` with `subprocess_args` from https://github.com/pyinstaller/pyinstaller/wiki/Recipe-subprocess, whereas e-dul's fix didn't work.  Moreover `subprocess_args` seems to be written in a way that is not platform specific.

**Comment by 2press** [comment]
#103

**Comment by int3l** [comment]
Thanks to @2press  this one is fixes by #103
